ui/admin/app/templates/scopes/scope/groups/group/add-members.hbs

```diff
@@ -30,62 +30,16 @@
 
   <page.body>
 
-    {{#let
-      (resource-filter 'scopes.scope.groups.group.add-members' 'scope')
-      as |filter|
-    }}
-      <Rose::Toolbar as |toolbar|>
-        <toolbar.primary>
-          <Rose::Form as |form|>
-            <Rose::Dropdown
-              @text={{t (concat 'resources.' filter.name '.title')}}
-              @count={{filter.selectedValue.length}}
-              @ignoreClickInside={{true}}
-              as |dropdown|
-            >
-              <ul>
-                <form.checkboxGroup
-                  @name={{filter.name}}
-                  @items={{this.flatSortedScopes}}
-                  @selectedItems={{filter.selectedValue}}
-                  @onChange={{fn this.callFilterBy filter.name}}
-                  as |group|
-                >
-                  <li
-                    class={{if
-                      group.item.isOrg
-                      'indent-label-1'
-                      (if group.item.isProject 'indent-label-2')
-                    }}
-                  >
-                    <dropdown.item>
-                      <group.checkbox
-                        @label={{group.item.displayName}}
-                        value={{group.item}}
-                      />
-                    </dropdown.item>
-                  </li>
-                </form.checkboxGroup>
-              </ul>
-            </Rose::Dropdown>
-
-            {{#if
-              (has-resource-filter-selections
-                'scopes.scope.groups.group.add-members' 'scope'
-              )
-            }}
-              <Rose::Button
-                @style='inline-link-action'
-                @iconLeft='flight-icons/svg/x-16'
-                {{on 'click' this.callClearAllFilters}}
-              >{{t 'actions.clear-all-filters'}}
-              </Rose::Button>
-            {{/if}}
-          </Rose::Form>
-        </toolbar.primary>
-      </Rose::Toolbar>
-    {{/let}}
-
+    <Dropdown
```

Comment:
Nice work @DhariniJeeva!

Curious - should we also be using the Hds::Dropdown component here?

Reply:
<Dropdown> lives in our core [addon](https://github.com/hashicorp/boundary-ui/blob/main/addons/core/addon/components/dropdown/index.hbs) which is HDS under the hood.

Reply:
fancy pants! 💅 👖